code/game/objects/items/devices/radio/electropack.dm

```diff
@@ -9,6 +9,7 @@
 	w_class = WEIGHT_CLASS_HUGE
 	materials = list(MAT_METAL=10000, MAT_GLASS=2500)
 	var/code = 2
+	var/sonic = 0

```

Comment:
I'm inclined to say this shoudl probably be named differently.

`noise_suppression` or something.

Also `FALSE`